toolchain/parse/tree.cpp

```diff
@@ -46,9 +46,9 @@ auto Tree::Parse(Lex::TokenizedBuffer& tokens, DiagnosticConsumer& consumer,
   context.AddLeafNode(NodeKind::FileEnd, *context.position());
 
   if (auto verify = tree.Verify(); !verify.ok()) {
-    if (vlog_stream) {
-      tree.Print(*vlog_stream);
-    }
+    // TODO: Restrict this to the subtree with the error, or only output
+    // when vlogging is enabled.
```

Comment:
```suggestion
    // TODO: This is temporarily printing to stderr directly during development. If we can, restrict this to a subtree with the error and add it to the stack trace (such as with PrettyStackTraceFunction). Otherwise, switch back to vlog_stream prior to broader distribution so that end users are hopefully comfortable copy-pasting stderr when there are bugs in tree construction.
```

Trying to offer some more information for the TODO so that someone doesn't come along and just change it to vlog.